packages/superset-ui-preset-chart-xy/src/encodeable/AxisAgent.ts

```diff
@@ -54,8 +54,21 @@ export default class AxisAgent<Def extends ChannelDef<Output>, Output extends Va
     return this.format || this.channelEncoder.formatValue;
   }
 
+  hasTitle() {
+    return this.getTitle() !== undefined;
+  }
+
   getTitle() {
-    return this.config.title || this.channelEncoder.getTitle();
+    if (typeof this.config.title === 'undefined' || this.config.title === true) {
+      return this.channelEncoder.getTitle();
+    }
+
+    const { title } = this.config;
+    if (title === 'false' || title === false || title === '') {
```

Comment:
it is a bit weird that we need to test `title === `false``. @kristw what is the reason behind it?

Reply:
agree. will remove this. storybook knobs didn't pass me boolean as expected.